docs/src/enterprise/getstarted/migrate-from-fluffy.md

```diff
@@ -0,0 +1,316 @@
+            external_user_id_claim_name: "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name" # This field was moved here!
+            validate_id_token_claims:
+              department: r_n_d
+          providers:
+            saml:
+              # enabled: true  // This field was dropped! 
+              sp_root_url: https://lakefs.company.com
+              sp_x509_key_path: dummy_saml_rsa.key
+              sp_x509_cert_path: dummy_saml_rsa.cert
+              sp_sign_request: true
+              sp_signature_method: "http://www.w3.org/2001/04/xmldsig-more#rsa-sha256"
+              idp_metadata_url: "https://adfs-auth.company.com/federationmetadata/2007-06/federationmetadata.xml"
+              post_login_redirect_url: / # This field was moved here!
+          ui_config:
+            login_url: "https://lakefs.company.com/sso/login-saml"
+            logout_url: "https://lakefs.company.com/sso/logout-saml"
+            login_cookie_names:
+              - internal_auth_session
+              - saml_auth_session
+        ```
+
+
+!!! note "OIDC"
+   
+    === "lakeFS + Fluffy (old)"
+        
+        ```yaml
+        # fluffy.yaml
+        auth:
+          post_login_redirect_url: /
+          logout_redirect_url: https://oidc-provider-url.com/logout/url
+          oidc:
+            enabled: true
+            url: https://oidc-provider-url.com/
+            client_id: <oidc-client-id>
+            client_secret: <oidc-client-secret>
+            callback_base_url: https://lakefs.company.com
+            is_default_login: true
+            logout_client_id_query_parameter: client_id
+            logout_endpoint_query_parameters:
+            - returnTo 
+            - https://lakefs.company.com/oidc/login
+        ```
+    
+        ```yaml
+        # lakefs.yaml
+        auth:
+          oidc:
+            friendly_name_claim_name: "name"
+            persist_friendly_name: true
+            default_initial_groups: ["Developers"]
+          ui_config:
+            login_url: /oidc/login
+            logout_url: /oidc/logout
+            login_cookie_names:
+            - internal_auth_session
+            - oidc_auth_session
+        ```
+
+    === "lakeFS Enterprise (new)"
+        
+        ```yaml
+        # lakefs.yaml
+        auth:
+          logout_redirect_url:  https://oidc-provider-url.com/logout/url
+          ui_config:
+            login_url: /oidc/login
+            logout_url: /oidc/logout
+            login_cookie_names:
+              - internal_auth_session
+              - oidc_auth_session
+          auth_source:
+          oidc:
+            friendly_name_claim_name: "nickname"
+            default_initial_groups: ["Admins"]
+          providers:
+            oidc:
+              # enabled: true  // This field was dropped!
+              post_login_redirect_url: http://localhost:8000/ # This field was moved here!
+              url: https://lakefs-cloud-dev.us.auth0.com/
```

Comment:
Be consistent with values you use `https://oidc-provider-url.com`  and now this? stay with the oidc-provider-url